### ISSUE TITLE
FAIL_INVALID error in nightly regression

### ISSUE THREAD

**Issue by ljianghedera** [hashgraph/hedera-services#470]
**Summary of the defect**
Observed the FAIL_INVALID errors again in nightly regression tests.

**How to reproduce (if possible)**
Check with CirclrCi's nightly regression summary URLs. For example, https://app.circleci.com/pipelines/github/hashgraph/hedera-services/2617/workflows/e4098ee0-0ec9-4511-a24c-b1cfe3ee9d1e/jobs/28551

**Service logs (if applicable)**
```

2020-08-22 09:54:06.824 INFO   125  ProviderRun - 40 minutes left in test - 27030 ops submitted so far (0 pending).
2020-08-22 09:54:06.824 INFO   132  ProviderRun - Precheck txn status counts :: {OK=11045}
2020-08-22 09:54:06.824 INFO   133  ProviderRun - Resolved txn status counts :: {INVALID_SIGNATURE=560, SUCCESS=10244, FILE_DELETED=237}
2020-08-22 09:54:06.824 INFO   134  ProviderRun - 
------------------------------

java.lang.AssertionError: Answer-only precheck was FAIL_INVALID, not one of [OK, INSUFFICIENT_TX_FEE, BUSY, INVALID_FILE_ID, PLATFORM_TRANSACTION_NOT_CREATED, FILE_DELETED]!
	at org.junit.Assert.fail(Assert.java:88)
	at com.hedera.services.bdd.spec.queries.HapiQueryOp.submitOp(HapiQueryOp.java:134)
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:164)
	at com.hedera.services.bdd.spec.utilops.grouping.ParallelSpecOps.lambda$submitOp$1(ParallelSpecOps.java:51)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1799)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
2020-08-22 09:54:07.903 WARN   180  HapiSpecOperation - 'UmbrellaRedux' - ParallelSpecOps{numSubOps=10} failed!
java.lang.AssertionError: Problem(s) with sub-operation(s): HapiGetFileContents{sigs=1, payer=uniquePayerAccount, node=0.0.3, file=file1479-RandomFile} :: Answer-only precheck was FAIL_INVALID, not one of [OK, INSUFFICIENT_TX_FEE, BUSY, INVALID_FILE_ID, PLATFORM_TRANSACTION_NOT_CREATED, FILE_DELETED]!
	at org.junit.Assert.fail(Assert.java:88) ~[junit-4.12.jar:4.12]
	at com.hedera.services.bdd.spec.utilops.grouping.ParallelSpecOps.submitOp(ParallelSpecOps.java:64) ~[classes/:?]
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:164) ~[classes/:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:38) ~[classes/:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[classes/:?]
	at com.hedera.services.bdd.spec.utilops.ProviderRun.submitOp(ProviderRun.java:147) ~[classes/:?]
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:164) ~[classes/:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.exec(HapiApiSpec.java:216) ~[classes/:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.run(HapiApiSpec.java:176) ~[classes/:?]
	at java.util.AbstractList$RandomAccessSpliterator.forEachRemaining(AbstractList.java:720) ~[?:?]
	at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:658) ~[?:?]
	at com.hedera.services.bdd.suites.HapiApiSuite.runSync(HapiApiSuite.java:233) ~[classes/:?]
	at com.hedera.services.bdd.suites.HapiApiSuite.runSuite(HapiApiSuite.java:125) ~[classes/:?]
	at com.hedera.services.bdd.suites.HapiApiSuite.runSuiteSync(HapiApiSuite.java:118) ~[classes/:?]
	at com.hedera.services.bdd.suites.SuiteRunner.lambda$runSuitesSync$14(SuiteRunner.java:352) ~[classes/:?]
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:176) ~[?:?]
	at java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948) ~[?:?]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[?:?]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) ~[?:?]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) ~[?:?]
	at com.hedera.services.bdd.suites.SuiteRunner.runSuitesSync(SuiteRunner.java:353) ~[classes/:?]
	at com.hedera.services.bdd.suites.SuiteRunner.lambda$runTargetCategories$10(SuiteRunner.java:326) ~[classes/:?]
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195) [?:?]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1654) [?:?]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) [?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) [?:?]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) [?:?]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) [?:?]
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) [?:?]
	at com.hedera.services.bdd.suites.SuiteRunner.runTargetCategories(SuiteRunner.java:326) [classes/:?]
	at com.hedera.services.bdd.suites.SuiteRunner.runCategories(SuiteRunner.java:295) [classes/:?]
	at com.hedera.services.bdd.suites.SuiteRunner.main(SuiteRunner.java:235) [classes/:?]
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:254) [exec-maven-plugin-3.0.0.jar:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]


```
**Environment:**
- OS: [e.g. Ubuntu 18.04]
- Java: [e.g. OpenJDK 11.0.4]
- Hedera Services Version: [e.g. 0.0.5]
- HAPI Version: [e.g. 0.0.5]

**Additional Context**
Add any other context about the problem here. Attach any logs here, if applicable.

**Comment by ljianghedera** [comment]
In the HAPI API, the description of FAIL_INVALID:


Name | Number | Description
-- | -- | --
FAIL_INVALID | 23 | There was a system error and the transaction failed because of invalid request parameters.

This error message doesn't  make much sense and is confusing. If the failure is because of "invalid request parameters",  we should tell the client what error(s) in the  parameters, and there shouldn't be any system error mentioned. If it's caused by system error, we should figure out what it is and probably to tell the client to re-try or take some other actions.

Simply put, it would be better to split this into two different error codes.

**Comment by noshmody** [comment]
Update the description/comment to reflect code.
